readme.md

```diff
@@ -7,51 +7,27 @@ InVEST models, and to be extensible to future models or common auxilary
 workflows of an InVEST user.
 
 ## To develop and launch this Application
-
-* `npm install`  
-* bind to an `invest` executeable (see package.json "invest" for a compatible version)
-
-In production, the invest exe comes from prebuilt binaries that are an artifact of the `invest` build process.  
-
-For development, duplicate the production setup by fetching prebuilt binaries:  
-`npm run fetch-invest`.  
-Or build your own local invest binaries using invest's `make binaries` and then
-copy the entire `dist/invest/` dir to this repo's `build/invest/`.  
-
+* `npm install`
+* `npm run fetch-invest`
+	+ fetches prebuilt invest binaries (see package.json invest property)
+	+ Alternatively, build your own local invest binaries:
+		* use invest's `make binaries`, then
+		* `cp -r invest/dist/invest/ invest-workbench/build/invest/`
+* `npm run dev` (this process stays live, do it in a separate shell)

```

Comment:
Okay, so with `npm run dev` and `npm start` below, we'll have two shells hanging out. Is that right?

Reply:
Yes, that's right